rllib/algorithms/ddpg/tests/test_ddpg.py

```diff
@@ -125,7 +130,8 @@ def test_ddpg_exploration_and_with_random_prerun(self):
 
     def test_ddpg_loss_function(self):
         """Tests DDPG loss function results across all frameworks."""
-        config = ddpg.DDPGConfig()
+        config = ddpg.DDPGConfig().training(num_steps_sampled_before_learning_starts=0)
```

Comment:
We now have to add this to a couple of tests. Before, the training iteration function would just run through with an empty sample batch if the buffer was not yet "ready".  With the changes to our execution plans, we now would wait for a while for enough samples to be collected with is nonsense for the compilation tests.